.buildkite/scripts/steps/test/ftr_configs.sh

```diff
@@ -52,6 +52,15 @@ while read -r config; do
     continue;
   fi
 
+  echo "--- Checking for existing scout event for config $config"
+  # We do not use --dontFailOnError here because we need a real exit code to determine if we should skip
+  if node scripts/scout check-event --buildNumber "${BUILDKITE_BUILD_NUMBER:-}" --testConfig "$config"; then
+    echo "--- Scout event found for config $config, skipping"
```

Comment:
These generate a lot of noise in the logs. I would probably log something like:
```
FULL_COMMAND="node scripts/functional_tests --bail --config $config $EXTRA_ARGS"

set +e # to avoid the script quitting
CHECK_EVENT_LOGS=$(node scripts/scout check-event --buildNumber "${BUILDKITE_BUILD_NUMBER:-}" --testConfig "$config")
CHECK_EVENT_RESULT=$?
set -e # turn back exit on error

if [[ "$CHECK_EVENT_RESULT" == "0" ]]; then
  echo "--- [ already-tested ] $FULL_COMMAND"
  echo "$CHECK_EVENT_LOGS" # so that the check event's logs are wrapped under the above banner
else
  echo "--- $ $FULL_COMMAND"
...
```